pkg/api/object.go

```diff
@@ -49,3 +52,31 @@ func (h *Handler) RemoveObjectStore(w http.ResponseWriter, r *http.Request) {
 	logger.Debugf("started async deletion of the object store")
 	w.WriteHeader(http.StatusAccepted)
 }
+
+// Gets connection information to the object store in this cluster.
+// GET
+// /objectstore/connectioninfo
+func (h *Handler) GetObjectStoreConnectionInfo(w http.ResponseWriter, r *http.Request) {
+	// TODO: auth is extremely important here because we are returning RGW credentials
+
+	// TODO: support other types of connection info to object store (such as swift)
```

Comment:
For the first one (auth) we do have a ticket for it #209, but I like calling out specific places in the codebase where we'll need to make updates with a fix down the road.  A bookmark if you will :)

I've opened #255 for the supporting of other connection types.